docker/nginx/entrypoint/30-letsencrypt-renew.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
```

Comment:
it looks copied. Please provide link to source and license information

Reply:
Bit this one is not. I wrote it from scratch :)